service/src/intTest/java/uk/nhs/adaptors/gp2gp/common/task/TaskQueueToDefinitionFactoryTest.java

```diff
@@ -0,0 +1,133 @@
+package uk.nhs.adaptors.gp2gp.common.task;
```

Comment:
This class is already unit tested. Writing an integration test is just duplicating test coverage and increasing the time/cost of maintaining the codebase.